lib/messages.js

```diff
@@ -22,33 +22,39 @@ var Messages = {
     var storedMessages = this.getStored();
     this.clearList();
     for (var i = 0; i < storedMessages.length; i++) {
-      this.render(storedMessages[i].text);
+      this.render(storedMessages[i]);
     }
   },
 
   clearList: function() {
     $messageList.html('');
   },
 
-  render: function(messageOutput) {
-    $messageList.append(`
-      <article class="message-js">
-        <p class="user-message" contenteditable="true">` + messageOutput + `</p>
-        <button class="delete-button-js"></button>
-      </article>
+  render: function(messageObject) {
+    if (messageObject.user === 'user') {
+      $messageList.append(`
+        <article class="message-js" id="${messageObject.id}">
```

Comment:
To identify the specific message when we search for the it.
On Sat, Aug 20, 2016 at 8:03 AM kylem038 <notifications@github.com> wrote:

> In lib/messages.js
> <https://github.com/blakeworsley/chatroom-adventures/pull/26#discussion_r75579079>
> :
>
> >      }
> >    },
> >
> >    clearList: function() {
> >      $messageList.html('');
> >    },
> >
> > -  render: function(messageOutput) {
> > -    $messageList.append(`
> > -      <article class="message-js">
> > -        <p class="user-message" contenteditable="true">` + messageOutput + `</p>
> > -        <button class="delete-button-js"></button>
> > -      </article>
> > +  render: function(messageObject) {
> > +    if (messageObject.user === 'user') {
> > +      $messageList.append(`
> > +        <article class="message-js" id="${messageObject.id}">
>
> Is there a reason why we need the message.id in the html?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/blakeworsley/chatroom-adventures/pull/26/files/8b31eee28550366de4f42fa41214dbe622c95ecb#r75579079>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG0p8xFZJrq0v35MLjcbKncVHOI0fknPks5qhwlPgaJpZM4JpAWD>
> .
>